### PR TITLE
You Can No Longer Remove The Inferno Part Of Disco Inferno

### PR DESCRIPTION
**Pull request by ghost**
## About The Pull Request
Makes the plasma floors on disco inferno a subtype of plasma floors that can't be crowbarred.

## Why It's Good For The Game
What if you couldn't destroy the gimmick of an entire shuttle with a crowbar

## Changelog
:cl:
add: subtype of plasma floors that can't be crowbarred
tweak: replaced all the floors on disco inferno with these new crowbar proof floors
/:cl:

### CODE REVIEW

File: code/game/turfs/open/floor/mineral_floor.dm [hunk 1]
```diff
@@ -58,6 +58,17 @@
 	if(exposed_temperature > 300)
 		PlasmaBurn(exposed_temperature)
 
+//Plasma floor that can't be removed, for disco inferno
+
+/turf/open/floor/mineral/plasma/disco
+	name = "plasma floor"
+	icon_state = "plasma"
+	floor_tile = /obj/item/stack/tile/mineral/plasma
+	icons = list("plasma","plasma_dam")
```
Comment (Fikou) [inline code/game/turfs/open/floor/mineral_floor.dm:67]: youre just repeating stuff from the parent, might as well just remove this whole segment

Comment (Paxilmaniac) [reply code/game/turfs/open/floor/mineral_floor.dm:67]: Very good point

**Comment by actioninja** [comment]
should just make it so if you try to crowbar it immediately ignites lul

**Comment by TemporalOroboros** [comment]
> 
> 
> should just make it so if you try to crowbar it immediately ignites lul

I can already hear the incoming bwoink for the poor soul who tries to be smart.

**Review by Cyberboss — APPROVED** [review]

**Comment by tgstation-server** [comment]
Your Fix/Feature pull request delta is currently below zero (-2). Maintainers may close future Feature/Tweak/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.

**Comment by nemvar** [comment]
Based robot

**Comment by TheSilverNuke** [comment]
I had no idea that was a thing. Imagine bugfixing.

**Pull request merged**